include/common/aarch32/asm_macros.S

```diff
@@ -79,6 +79,16 @@
 	ldr r0, =(\_name + \_size)
 	.endm
 
+#if (ARM_ARCH_MAJOR == 7)
+	/* ARMv7 does not support stl instruction */
+	.macro stl _reg, _write_lock
+	dmb
+	str	\_reg, \_write_lock
+	dsb
+	sev
```

Comment:
minor: Not sure whether the `sev` is needed.

Reply:
Edit: When this instruction is used in  spin lock context "In ARMv8, when the global monitor for a PE changes from Exclusive Access state to Open Access state, an event is generated."

Whereas on ARMv7 this is not the case.  `sev` is required for ARMv7 to match the `wfe` in spin_lock().

It would be better to isolate this change to the aarch32 spin_lock.S file similar to the aarch64 `COND_SEV()` implementation.